required/offline_API_wrapper.js

```diff
@@ -68,7 +81,18 @@ var API = {
 	LMSStore: function(force) {
 		if (window.ISCOOKIELMS === false) return;
 		if (!force && API.cookie("_spoor") === undefined) return;
-		API.cookie("_spoor", JSON.stringify(this.data));
+
+		var stringified = JSON.stringify(this.data);
+
+		API.cookie("_spoor", stringified);
+
+		// a length mismatch will most likely indicate cookie storage limit exceeded
+		if (API.cookie("_spoor").length != stringified.length) {
```

Comment:
@tomgreenfield no, you were correct, we still want to show the error. The if statements are just concerned with looking for `_suppressErrors` and if set to true we use `console.error` rather than the more in your face `alert`